app/models/like.rb

```diff
@@ -0,0 +1,7 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post, counter_cache: true
+  def update_counter
+    post.update(likes_counter: post.likes.size)
+  end
+end
```

Comment:
- [ ] Finally here again, It would be nice to use one of the Rails active callbacks after_save to call the #update_counter method as suggested in the Post model comment above. Kindly do so.